repo/packages/C/crate/0/config.json

```diff
@@ -0,0 +1,156 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+
+    "crate": {
+      "description": "Crate configuration properties.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "string",
+          "description": "The version number of Crate that should be used for the framework."
+        },
+        "cluster-name": {
+          "type": "string",
+          "description": "The name of the Crate cluster.",
+          "default": "crate"
+        },
+        "http-port": {
+          "type": "integer",
+          "description": "HTTP port of the Crate instances.",
+          "default": 4200,
+          "minimum": 0,
+          "maximum": 65535
+        },
+        "transport-port": {
+          "type": "integer",
+          "description": "Internal transport port of the Crate instances.",
+          "default": 4300,
+          "minimum": 0,
+          "maximum": 65535
+        },
+        "data-path": {
+          "type": "string",
+          "description": "The absolute path where Crate should store table data. Note that the user `crate` needs write access!"
+        },
+        "blob-path": {
+          "type": "string",
+          "description": "The absolute path where Crate should store blobs. Note that the user `crate` needs write access!"
+        },
+
+        "framework": {
```

Comment:
we need a property named framework-name to be defined.

example: https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/M/memsql/0/config.json#L47

also please use the framework-name for the id... I'll comment there.